src/formats/mmCIF.cpp

```diff
@@ -122,16 +122,15 @@ void mmCIFFormat::init_() {
                     trimmed.substr(1, trimmed.size() - 2).to_string() : "";
         }
 
-        if (in_loop && line_split[0].find("_atom_site") != std::string::npos) {
+        if (in_loop && line_split[0].find("_atom_site.") != std::string::npos) {
             auto atom_label = line_split[0].substr(11).to_string();
-            to_ascii_lowercase(atom_label);
```

Comment:
Why is this removed?

Reply:
Just to make the code match the files. If the file uses `Cartn_x` or `pdbx_PDB_model_num`, I'd rather have the same strings in the code, without case change. Should also prevent reading data from the wrong entry if the file is malformed and contains both `cartn_x` and `Cartn_x`.